pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.onarandombox.multiversecore</groupId>
     <artifactId>Multiverse-Core</artifactId>
-    <version>3.0.0-SNAPSHOT</version>
+    <version>3.1.0-SNAPSHOT</version>
```

Comment:
Heh, I don't think we've ever bumped the version. I'll leave this up to @dumptruckman, but I'd think something as trivial as removing/updating for a version should just go with the model we've had for years (while it may not be the best).